### PR TITLE
SG-123 fix quantity and logo

### PR DESCRIPTION
**Pull request by augsteyer**
### Changed logo:
image

### Fixed issue with string -> int:

image

The original error:
```
[{"errors":[{"code":"INVALID_TYPE","message":"Expected type number but found type string","path":["orders","0","lineItems","0","quantity"]}],"warnings":[]}}}
```

### CODE REVIEW

**Comment by augsteyer** [comment]
@Rekhyt I cannot assign you on this one, so you are it! :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Changed
+- to new Shopgate logo
```
Comment (Rekhyt) [inline CHANGELOG.md]:
```suggestion
- replaced retail.red logo with the new one from Shopgate
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
...
+### Fixed
+- product quantity change was providing data as a string instead of the required integer
```
Comment (Rekhyt) [inline CHANGELOG.md]:
```suggestion
- product quantity sent as a string to the API, number required
```

**Review by Rekhyt — CHANGES_REQUESTED** [review]

**Comment by augsteyer** [comment]
Apparently I cannot batch commit for some reason :/

**Review by Rekhyt — APPROVED** [review]

**Comment by Rekhyt** [comment]
> Apparently I cannot batch commit for some reason :/

np, merged, thank you for the PR :+1:

**Pull request merged**